app/src/main/java/net/skyscanner/backpack/demo/compose/CardButtonsStory.kt

```diff
@@ -0,0 +1,148 @@
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButton
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonSize
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonStyle
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonType
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkDimension
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun CardButtonsStory(
+  size: BpkCardButtonSize,
+  modifier: Modifier = Modifier,
+) {
+  LazyColumn(
+    modifier
+      .fillMaxSize()
+      .padding(top = BpkDimension.Spacing.Md)
+  ) {
+    item {
+      Row(
+        modifier = modifier.background(BpkTheme.colors.textOnDark).fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceEvenly
+      ) {
+        var checked1 by remember { mutableStateOf(false) }
+        BpkCardButton(
+          type = BpkCardButtonType.Save(checked = checked1, onCheckedChange = { checked1 = !checked1 }),
+          size = size,
+          style = BpkCardButtonStyle.Default,
+          contentDescription = "",
+        )
+        var checked2 by remember { mutableStateOf(true) }
+        BpkCardButton(
+          type = BpkCardButtonType.Save(checked = !checked2, onCheckedChange = { checked2 = !checked2 }),
+          size = size,
+          style = BpkCardButtonStyle.Default,
+          contentDescription = "",
+        )
+        BpkCardButton(
+          type = BpkCardButtonType.Share(onClick = {}),
+          size = size,
+          style = BpkCardButtonStyle.Default,
+          contentDescription = "",
+        )
+      }
+    }
+    item {
+      Row(
+        modifier = modifier.background(BpkTheme.colors.surfaceContrast).fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceEvenly
+      ) {
+        var checked1 by remember { mutableStateOf(false) }
+        BpkCardButton(
+          type = BpkCardButtonType.Save(checked = checked1, onCheckedChange = { checked1 = !checked1 }),
+          size = size,
+          style = BpkCardButtonStyle.OnDark,
+          contentDescription = "",
+        )
+        var checked2 by remember { mutableStateOf(true) }
+        BpkCardButton(
+          type = BpkCardButtonType.Save(checked = !checked2, onCheckedChange = { checked2 = !checked2 }),
```

Comment:
is the `!checked` needed in the initial state? looking at the screenshots both saved & unsaved options are unchecked